app/services/computacenter_ledger_csv.rb

```diff
@@ -0,0 +1,80 @@
+require 'csv'
+
+class ComputacenterLedgerCsv
```

Comment:
As above, where we have a `Computacenter` prefix, could this actually be in the `Computacenter::` namespace (& corresponding sub-directory) instead?

Also, having a `to_csv` method on something which is already a `(something)Csv` feels awkward. 
How about calling this just `Computacenter::Ledger`? 
You'd then be initializing the ledger with some users - `ledger = Computacenter::Ledger.new(users: some_users)` - and generating CSV with `ledger.to_csv` ?

Naming things _is_ hard :)